compiler/src/dmd/lexer.d

```diff
@@ -3019,7 +3019,10 @@ class Lexer
         auto dc = (lineComment && anyToken) ? &t.lineComment : &t.blockComment;
         // Combine with previous doc comment, if any
         if (*dc)
-            *dc = combineComments(*dc, buf[], newParagraph).toDString();
+        {
+            auto p = combineComments(*dc, buf[], newParagraph);
+            *dc = p ? p[0 .. strlen(p)] : null;
```

Comment:
`toDString` does this itself, so the change seems to make the code worse

Reply:
I did it because it is trivial, and does not drag in root.string which includes a bunch of other stuff.

Reply:
But `root.string` is a leaf module, so it's fine to drag it in.